pytorch_pfn_extras/training/extensions/print_report.py

```diff
@@ -142,7 +139,7 @@ def _update_entries(self, log_report: log_report_module.LogReport) -> None:
 
     def __call__(self, manager: _BaseExtensionsManager) -> None:
         log_report = self.get_log_report(manager)
-        log = log_report.log
+        log_report._log.register_looker('print_report')
```

Comment:
Is it too late to register?

Reply:
It will be resolved by setting `call_before_training=True` by default. I will fix it after #349 is merged.

Reply:
I will move `register_looker` to `initialize` method instead.